packages/core/src/accordion/AccordionHeader.css

```diff
@@ -6,37 +6,47 @@
   align-items: center;
   border: 0;
   width: 100%;
-  min-height: var(--salt-size-base);
+  min-height: calc(var(--salt-size-base) + var(--salt-spacing-100));
   background: var(--salt-actionable-secondary-background);
   font-size: var(--salt-text-fontSize);
   line-height: var(--salt-text-lineHeight);
   font-family: var(--salt-text-fontFamily);
   color: var(--salt-text-primary-foreground);
-  --saltIcon-color: var(--salt-text-primary-foreground);
+  font-weight: var(--salt-text-fontWeight-strong);
+  --saltIcon-color: currentcolor;
```

Comment:
Seeing this more often these days makes me think whether this should be the default

Reply:
I have already brought it up, but haven't been able to chat with a designer